engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeMethodNode.java

```diff
@@ -210,6 +210,30 @@ Object doFunctionalDispatchUncachedSymbol(
     Type selfTpe = typesLibrary.getType(self);
     Function function = resolveFunction(symbol, selfTpe, methodResolverNode);
     if (function == null) {
+      if (self instanceof Type t && isNamedAsAssociatedType(t)) {
+        CompilerDirectives.transferToInterpreter();
+        // if we represent a Module
+        var scope = t.getDefinitionScope();
+        var optionType = scope.getType(symbol.getName());
+        if (optionType.isPresent()) {
+          return optionType.get();
+        }
+        var module = scope.getModule();
+
+        var ctx = EnsoContext.get(this);
+        var moduleName =
+            module.getName().item().equals("Main")
+                ? module.getPackage().libraryName().toString()
+                : module.getName().toString();
+        var subModuleName = moduleName + "." + symbol.getName();
+        var optionModule = ctx.getTopScope().getModule(subModuleName);
+        if (optionModule.isPresent()) {
+          var subType = optionModule.get().getScope().getAssociatedType();
+          if (subType != null) {
+            return subType;
```

Comment:
I have no idea how can this works. In this control-flow-path, we are not executing anything, we are just returning a Type, but this Java method should execute some Enso method.  I still have suspicion that this can break for some FQN method invocations. I will try to come up with some failing tests to see if my suspicion is correct.

Reply:
When we invoke `Delimited_Format` on object representing `Standard.Table`, then 

> we are not executing anything, we are just returning a Type

we are supposed to return the same thing as `Standard.Table.Delimited_Format`. Which means to _execute nothing_ and _return the type_. Exactly as you say.

> come up with some failing tests to see if my suspicion is correct.

More tests is always welcomed, but I'd like to remind that we have already burned too much time on #9367 and there are more urgent things to fix once this PR gets merged and there is no known example of _FQN and imports_ inconsistency.